site/index.html

```diff
@@ -54,6 +54,22 @@ <h2 class="event-title">Next Event</h2>
             </article>
           </div>
         </div>
+          <secion class="image-gallery">
```

Comment:
Also, this image gallery should be outside/below the `<div class="row"` that is containing the two columns above it.